src/lib/hooks/use-best-quote/utils/issuance/issuance-quote.ts

```diff
@@ -27,15 +32,29 @@ interface IssuanceQuoteRequest {
   slippage: number
 }
 
+function getOutputTokenAmount(
+  isIssuance: boolean,
+  outputToken: Token,
+  inputTokenAmount: bigint,
+  units: bigint,
+): bigint {
+  if (!isIssuance) return units
+  if (outputToken.symbol !== RealWorldAssetIndex.symbol) return units
+  return (
+    (inputTokenAmount * BigInt('1000000000000000000')) /
+    BigInt('23800000000000000')
```

Comment:
For issuances we need to take the exchange rate into account - as otherwise the output token amount would be wrong since RWA is not 1-1 wstETH like hyETH before.